core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -689,6 +696,54 @@ public BlockInfo getBlockInfo(long blockId) throws BlockInfoException, Unavailab
     return ret;
   }
 
+  private MasterWorkerInfo lookupWorker(WorkerNetAddress workerNetAddress) {
+    for (IndexedSet<MasterWorkerInfo> workers: Arrays.asList(mTempWorkers, mLostWorkers)) {
+      MasterWorkerInfo worker = workers.getFirstByField(ADDRESS_INDEX, workerNetAddress);
+      if (worker != null) {
+        return worker;
+      }
+    }
+    return null;
+  }
+
+  private MasterWorkerInfo lookupWorker(long workerId) {
+    for (IndexedSet<MasterWorkerInfo> workers: Arrays.asList(mTempWorkers, mLostWorkers)) {
+      MasterWorkerInfo worker = workers.getFirstByField(ID_INDEX, workerId);
+      if (worker != null) {
+        return worker;
+      }
+    }
+    return null;
+  }
+
+  private MasterWorkerInfo moveWorker(long workerId) {
```

Comment:
- annotate this method by `@Nullable` please because null can be returned.
- I would name this method with more info, like `moveToRegisteredWorkers`

Reply:
done, changed as registerWorkerInternal